public/keymaps/r/ramonimbao_herringbone_default.json

```diff
@@ -0,0 +1,16 @@
+{
+  "keyboard": "ramonimbao/herringbone",
+  "keymap": "default",
+  "commit": "32a47e7af4a2ce21260aadde5bf7ed8f78293ff9",
+  "layout": "LAYOUT_ansi",
+  "layers": [
+    [
+      "KC_ESC",     "KC_F1", "KC_F2", "KC_F3", "KC_F4",     "KC_F5", "KC_F6", "KC_F7", "KC_F8",     "KC_F9", "KC_F10", "KC_F11", "KC_F12",  "KC_PSCR", "KC_DEL",
+      "KC_GRV", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_EQL",       "KC_BSPC",         "KC_HOME",
```

Comment:
General note here: the default keymap on the qmk_firmware has `KC_TILD` instead of `KC_GRV`. You should maybe fix that over there.

Everything else here looks good for the moment (though the commit hash will need updating if you fix the qmk_firmware keymap).

Reply:
Yep, it's pending the merge currently (https://github.com/qmk/qmk_firmware/pull/9578).

Right, I forgot that the hash would change. I'll update it when the PR gets merged.

Reply:
> Yep, it's pending the merge currently `qmk/qmk_firmware#9578`

Merged.

Reply:
Thanks! I've updated the hash as well.